design/resources.go

```diff
@@ -437,3 +437,283 @@ var _ = a.Resource("search", func() {
 		a.Response(d.InternalServerError)
 	})
 })
+
+var _ = a.Resource("work-item-link-category", func() {
+	a.BasePath("/workitemlinkcategories")
+
+	a.Action("show", func() {
+		a.Routing(
+			a.GET("/:id"),
+		)
+		a.Description("Retrieve work item link category (as JSONAPI) for the given ID.")
+		a.Params(func() {
+			a.Param("id", d.String, "ID of the work item link category")
+		})
+		a.Response(d.OK, func() {
+			a.Media(WorkItemLinkCategory)
+		})
+		a.Response(d.BadRequest, JSONAPIErrors)
+		a.Response(d.InternalServerError, JSONAPIErrors)
+		a.Response(d.NotFound, JSONAPIErrors)
+	})
+
+	a.Action("list", func() {
+		a.Routing(
+			a.GET(""),
+		)
+		a.Description("List work item link categories.")
+		a.Response(d.OK, func() {
+			a.Media(WorkItemLinkCategoryArray)
+		})
+		a.Response(d.BadRequest, func() {
```

Comment:
Yes, I've opened an issue at goa for this: https://github.com/goadesign/goa/issues/898